museum/demo/planning_recurrency.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="planning_recurrency_1" model="planning.recurrency">
+    <field name="repeat_unit">day</field>
+    <field name="last_generated_end_datetime">2026-01-02 09:00:00</field>
```

Comment:
Static dates? You can replace by relative

Reply:
There is now a test about, thanks to @abdrahmanrashed https://github.com/odoo/industry/commit/dfe126b80de7957f42e0927106db6a921a9134cd